.rubocop.yml

```diff
@@ -1,4 +1,4 @@
-require:
```

Comment:
Should we pin `rubocop` to 1.72 or greater in the gemspec? I am not sure if this syntax would work with older version of `rubocop`.